docs/appkit/shared/options.mdx

```diff
@@ -13,6 +13,21 @@ The following options can be passed to the `createAppKit` function:
 createAppKit({ adapters, projectId, networks, ...options })
 ```
 
+## networks
+
+You can set an array of networks that can be chosen from the library `@reown/appkit/networks`. This library get the list of viem networks and adds the Solana ones.
```

Comment:
```suggestion
You can set an array of networks that can be chosen from the `@reown/appkit/networks` library. This library retrieves the list of EVM networks from Viem and also includes the Solana networks.
```

Updating the text for correct grammar and mention that Viem only exposes EVM compatible networks.